kafka/consumer/go.sum

```diff
@@ -1,205 +1,55 @@
-github.com/Azure/azure-sdk-for-go/sdk/azcore v1.10.0 h1:n1DH8TPV4qqPTje2RcUBYwtrTWlabVp4n46+74X2pn4=
-github.com/Azure/azure-sdk-for-go/sdk/azcore v1.10.0/go.mod h1:HDcZnuGbiyppErN6lB+idp4CKhjbc8gwjto6OPpyggM=
```

Comment:
This looks like you used a `go mod clean` at some point?  In general, we don't want to use that so that we have rollback protection from malicious rollback attacks.